scripts/triggers-end-to-end-tests/run.spec.js

```diff
@@ -31,9 +31,9 @@ const FIRESTORE_FUNCTION_LOG = "========== FIRESTORE FUNCTION ==========";
  * parallel emulator subprocesses.
  */
 const TEST_SETUP_TIMEOUT = 20000;
-const EMULATORS_STARTUP_DELAY_MS = 7000;
-const EMULATORS_WRITE_DELAY_MS = 5000;
-const EMULATORS_SHUTDOWN_DELAY_MS = 2000;
+const EMULATORS_STARTUP_DELAY_MS = 10000;
```

Comment:
This is an unrelated change but the tests were about 50% flaky on this PR so I decided to just fix it.